### PR TITLE
geteltorito: init at 0.6

### PR DESCRIPTION
**Pull request by Profpatsch**
- Built on platform(s)
  - [x] NixOS
- [x] Tested execution of all binary files (usually in `./result/bin/`)

### CODE REVIEW

**Comment by Profpatsch** [comment]
Update: Doesn’t depend on fetchdir anymore, uses a github mirror repo instead.

**Comment by RamKromberg** [comment]
it's your own git... Just commit the .roff \ man page instead of depending on ronn :/

File: pkgs/tools/misc/geteltorito/default.nix [hunk 1]
```diff
@@ -0,0 +1,50 @@
+{ stdenv, perl, ronn, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  name = "geteltorito-${version}";
+  version = "0.5";
```
Comment (RamKromberg) [inline pkgs/tools/misc/geteltorito/default.nix]: 0.6 (https://github.com/Profpatsch/geteltorito/blob/master/geteltorito#L18)

Comment (Profpatsch) [inline pkgs/tools/misc/geteltorito/default.nix]: Wow, the readme still said 0.5. Very inconsistent.

**Comment by Profpatsch** [comment]
> it's your own git... Just commit the .roff \ man page instead of depending on ronn :/

It’s nice, but I’m not particularly fond of adding my own stuff to the repo. This means I’d have to manually adapt it each time upstream updates.

**Comment by RamKromberg** [comment]
Your call.

For reference:
```
{ stdenv, perl, fetchFromGitHub }:

stdenv.mkDerivation rec {
  name = "geteltorito-${version}";
  version = "0.6";

  src = fetchFromGitHub {
    owner = "Profpatsch";
    repo = "geteltorito";
    rev = version;
    sha256 = "05bcn3pam29xmsz1ykyqsdbkz8y23kcrvvhm987f65wd1g741f75";
  };

  nativeBuildInputs = [ perl ];
  dontBuild = true;
  geteltorito_mdoc = ''
    .Dd 2015-02-25
    .Dt GETELTORITO 1
    .Os Unix
    .Sh NAME
    .Nm geteltorito
    .Nd an El Torito boot image extractor
    .Sh SYNOPSIS
    .Nm
    .Op Fl hv
    .Op Fl o Ar outputfilename
    .Ar cd-image
    .Nm
    .Ar cd-image
    >
    .Ar outputfilename
    .Sh DESCRIPTION
    .Nm
    will try to extract the initial (default) El Torito image from a bootable CD (or cd-image) given by
    .Ar cd-image
    and write the data extracted to STDOUT or to
    .Ar outputfilename \.
    All other information is written to STDERR (e.g. type and size of image).
    .Pp
    .Nm
    will not extract any of other possibly existing bootimages that are allowed by the El Torito standard.
    .Pp
    .Bl -tag -width "outputfilename" -compact
    .It cd-image
    bootable CD or cd-image
    .It outputfilename
    optional output file name
    .El
    .Bl -tag -width -indent
    .It Fl h
    Build-in help.
    .It Fl v
    Print version of script and exit.
    .It Fl o
    Write extracted data to
    .Pa outputfilename
    instead of
    .Pa STDOUT \.
    .El
    .Sh ABOUT
    .Bl -tag -width "Version:"
    .It Author:
    Rainer Krienke
    .It Email:
    krienke@uni-koblenz.de
    .It License:
    GPL v2
    .It Version:
    0.6
    .El
    .Sh FILES
    .Bl -tag -width "/usr/share/man/man1/geteltorito.1" -compact
    .It Pa /usr/share/bin/geteltorito
    The
    .Nm
    perl script.
    .It Pa /usr/share/man/man1/geteltorito.1
    This help file.
    .El
    .Sh SEE ALSO
    .Xr perl 1

  '';

  installPhase = ''
    mkdir -p $out/man/man1
    echo "${geteltorito_mdoc}" > $out/man/man1/geteltorito.1
    install -vD geteltorito $out/bin/geteltorito
  '';

  meta = with stdenv.lib; {
    description = "Extract the initial/default boot image from a CD image if existent";
    homepage = "https://userpages.uni-koblenz.de/~krienke/ftp/noarch/geteltorito/";
    maintainer = with maintainers; [ profpatsch ];
    license = licenses.gpl2;
    platforms = with platforms; all;
  };

}

```

**Pull request merged**